tests/acceptance-tests/test/specs/smoke-tests/paystack.app-test.js

```diff
@@ -0,0 +1,51 @@
+const yaml = require("js-yaml");
+const fs   = require("fs");
+const expect = require("chai").expect;
+const getId = require("../../../lib/get-elements.js");
+const dotenv = require("dotenv");
+
+dotenv.config();
+beforeEach(function () {
+  const browserConfig = yaml.safeLoad(fs.readFileSync("./tests/acceptance-tests/config/settings.yml", "utf8"));
+  const baseUrl = browserConfig.base_url.toString();
+  browser.url(baseUrl);
+  // browser.getSession().then(function (sessionid) {
```

Comment:
why do you have commented-out code in source control?

Reply:
I will fix that right now.